app/src/components/report/MyModal.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useImperativeHandle, forwardRef } from 'react';
+import Box from '@mui/material/Box';
+import { Button } from 'react-bootstrap';
+import Typography from '@mui/material/Typography';
+import Modal from '@mui/material/Modal';
+
+const style = {
```

Comment:
Nit: it's conventional for constants to be named in `UPPER_SNAKE_CASE`, so this could be `STYLE`